backend/app/tests/api/routes/test_project.py

```
@@ -94,3 +95,48 @@ def test_delete_project(
+    )
+    assert response.status_code == 200
+    response_data = response.json()
+    assert "data" in response_data
+    assert isinstance(response_data["data"], list)
+    assert len(response_data["data"]) >= 1
+    assert any(p["id"] == project.id for p in response_data["data"])
+
+
+# Test retrieving projects by non-existent organization
+def test_read_projects_by_organization_not_found(
+    db: Session, superuser_token_headers: dict[str, str]
```

Ruff (0.15.7)

[warning] Unused function argument: `db`

(ARG001)